components/Org/OrgDetail.js

```diff
@@ -1,93 +1,137 @@
-import { Button, Icon, Tabs } from 'antd'
+import { Tabs, Divider } from 'antd'
 import Markdown from 'markdown-to-jsx'
 import Head from 'next/head'
 import PropTypes from 'prop-types'
 import React from 'react'
 import { FormattedMessage } from 'react-intl'
 import styled from 'styled-components'
 import MemberSection from '../Member/MemberSection'
-import { FullPage, H3Bold, PageHeaderContainer, PBold } from '../VTheme/VTheme'
-import OrgCategory from './OrgCategory'
-const ButtonGroup = Button.Group
+import RegisterMemberSection from '../Member/RegisterMemberSection'
+import { PageHeaderContainer } from '../VTheme/VTheme'
+import OrgDetailForm from '../Org/OrgDetailForm'
 
 function callback (key) {
   // TODO: [VP-300] on tab change update the path so that the page is bookmark and reloadable
 }
 const shadowStyle = { overflow: 'visible', textAlign: 'center' }
 const { TabPane } = Tabs
 
+const OrgPageWrapper = styled.div`
```

Comment:
Please remove this and apply the changes to FullPage.  There is no reason for OrgPage to behave differently to all the other pages.  Each page style should be defined only once and used everywhere, otherwise, we will waste a lot of time trying to get the pages consistent. 
Please Move these style elements to VTheme and agree on them with Walt before implementing changes to how a page looks.
I think a good rule is that there are no @media styles outside of VTheme components.  Page behaviour, Title styles and section wrappers MUST be consistent everywhere. 
So no local styles except for unusually placed components.

Reply:
I have changed this back to FullPage, I initially made my own component as I wanted to removed the large margin at the bottom of the FullPage component but did not want to make site wide changes (so I didn't want to touch FullyPage in Vtheme) and I only wanted it removed on the Orgpage for the time being. 

@media screen styles are used throughout the pages already. This is a good way to keep styled components responsive.